app/views/teacher_interface/application_forms/show/_declined.html.erb

```diff
@@ -1,17 +1,18 @@
 <h2 class="govuk-heading-l">Your QTS application has been declined</h2>
 
-<p class="govuk-body">Thank you for applying for qualified teacher status (QTS) and for your patience while we reviewed your application.</p>
+<p class="govuk-body">Thank you for applying for qualified teacher status (QTS) and for your patience while we reviewed your application. Unfortunately, your application has been declined.</p>
 
 <h3 class="govuk-heading-m">Reason for decline</h3>
 
 <% view_object.declined_reasons.each do |title, reasons| %>
   <% if title.present? %>
-    <h4 class="govuk-heading-s"><%= title %></h4>
+    <h3 class="govuk-heading-s"><%= title %></h3>
   <% end %>
 
   <% if reasons.size == 1 %>
     <%= simple_format reasons.first[:name] %>
     <% if reasons.first[:assessor_note].present? %>
+      <h3 class="govuk-heading-s"><%= "Assessor notes" %></h3>
```

Comment:
```suggestion
      <% if title.present? %>
        <h5 class="govuk-heading-s">Assessor notes</h5>
      <% else %>
        <h4 class="govuk-heading-s">Assessor notes</h4>
      <% end %>
```

This should be `h5` as it belongs within the `h4` of the section title, or a `h4` if there's no title as it belongs within the `h3` of "Reason for decline".